README.md

```diff
@@ -1 +1,32 @@
 # My INF 7420 Project
+
+I will be building a travel experience website. The purpose of this project is to share my travel experiences in a unique way to inspire viewers to connect with peoples and places of the world, and with nature
+
+## Wireframes
+
+I value planning and user experience so I have created wireframes to show the architecture and design of my website.
+
+![Wireframe of the Landing Page](wireframes/wireframe-homepage.jpg)
```

Comment:
References to the source are case sensitive so you will have to change this from `wireframe-homepage.jpg` to match the filename of the image which is `wireframe-homepage.JPG`. Same applies to the other images.